Components/button.css

```diff
@@ -0,0 +1,59 @@
+  
+  .btn-primary {
+    background-color: var(--primary-color);
+  }
+  
+  .btn-secondary {
+    background-color: var(--secondary-color);
+  }
+  
+  .btn-error {
+    background-color: var(--error-color);
+  }
+  
+  .btn-success {
+    background-color: var(--success-color);
+  }
+  
+  .btn-primary:hover {
+    background-color: var(--primary-color-hover);
+  }
+  
+  .btn-secondary:hover {
+    background-color: var(--secondary-color-hover);
+  }
+  
+  .btn-error:hover {
+    background-color: var(--error-color-hover);
+  }
+  
+  .btn-success:hover {
+    background-color: var(--success-color-hover);
+  }
+  
+  .btn-link {
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  
+  .btn-icon {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 5px;
+  }
+  
+  .floating-btn {
+    height: 2.5rem;
+    width: 2.5rem;
+    border-radius: 999px;
```

Comment:
don't use absolute units for such high values, use rem. If there's a reason let me know i may be wrong

Reply:
I saw 999px in the figma file that was shared and hence thought I'll keep it that way